src/GuestConfiguration/GuestConfiguration.Test/GuestConfiguration.Test.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.GuestConfiguration" Version="1.1.0" />
+    <PackageReference Include="Microsoft.Azure.Management.PolicyInsights" Version="2.0.2" />
```

Comment:
This is going to cause problems when the PolicyInsights client is updated by that team.  One option would be to move these cmdlets into the PolicyInsights module.  Another would be to capture a common library for policy insights here: https://github.com/Azure/azure-powershell-common/tree/master/src